### PR TITLE
Use fixed version of nodeify

### PR DESCRIPTION
**Pull request by mitar**
The 1.0.0 version has an ugly bug: https://github.com/then/nodeify/pull/2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "js-sha3": "^0.5.7",
     "multihashes": "~0.4.3",
     "murmurhash3js": "^3.0.1",
-    "nodeify": "^1.0.0"
+    "nodeify": "git+ssh://git@github.com/then/nodeify.git#444f8a68890fd08b9b30113784833283c61d277e"
```
Comment (tabrath) [inline package.json]: This doesn't work and breaks the build. Should be git://github.com/then/nodeify.git#444f8a68890fd08b9b30113784833283c61d277e

**Comment by mitar** [comment]
Changed.

**Comment by mitar** [comment]
`1.0.1` version has been released.

**Comment by dignifiedquire** [comment]
Thank you :octocat:

**Pull request merged**